docs/Developers/Design/crd_design.md

```diff
@@ -0,0 +1,58 @@
+# Custom Resource Definitions
+
+## **StorageCatalog**
+
+### *Singular: StorageCatalog, Plural: StorageCatalogs*
+
+```YAML
+kind: StorageCatalog
+spec:
+  - storage: "ceph"
+    version: ["mimic", "nautilus"]
```

Comment:
Do we want to support mimic here?

Reply:
This is just an example to show what the values could be.

Reply:
Can the version field be numbers (3.2 or 4.5.2 etc) instead of labels?

Reply:
Yes.